src/eureka/S5_lightcurve_fitting/differentiable_models/StarryModel.py

```diff
@@ -185,8 +195,51 @@ def setup(self):
             planet.theta0 = 180.0
             planet.t0 = temp.t0
 
+            # Pixel sampling setup
+            if 'pixel_ydeg' in self.paramtitles:
+                # Get pixel transform matrix and number of pixels
+                if self.ydeg == 2:
+                    self.oversample = 3
+                else:
+                    self.oversample = 2
+
+                A = planet.map.get_pixel_transforms(
+                    oversample=self.oversample)[3]
+                self.npix = A.shape[1]
+
+                # Set prior to either be log normal, or normal around zero
+                if self.force_positivity:
+                    p = pm.LogNormal("p", mu=np.log(0.3*amp), tau=1.0,
+                                     shape=(self.npix,))
```

Comment:
Similarly, is this prior really broad enough to be minimally informative and be hard coded?

Reply:
See above comment, this is now settable in the run parameters file